Sources/SwiftUINavigationCore/Documentation.docc/Articles/Dialogs.md

```diff
@@ -0,0 +1,101 @@
+# Dialogs
```

Comment:
We already have an `AlertsDialogs.md` that is a good place for docs related to presenting alerts and dialogs. Was this added by mistake? Also, feel free to omit new DocC articles for now. We can consider updating existing docs closer to a version release :smile:

Reply:
If preserving, what would be the best approach to make it consistent?

Given there isn't really any other documentation in `Core` currently I think I'll just remove the new articles.